assembly/assembly-machine-war/src/main/java/org/eclipse/che/wsagent/server/WsAgentModule.java

```diff
@@ -68,10 +67,9 @@ protected void configure() {
         install(new ArchetypeGeneratorModule());
         install(new GitHubModule());
         install(new org.eclipse.che.swagger.deploy.DocsModule());
+        install(new DebuggerModule());
```

Comment:
I think here should be full fqn:  install(new org.eclipse.che.api.debugger.server.DebuggerModule());